docker/maistra-builder_2.1-1.9.Dockerfile

```diff
@@ -0,0 +1,148 @@
+FROM fedora:33
```

Comment:
can we use f34?

Reply:
I'm using f33 here because it has the right golang version (1.15.x). I'm open to updating to f34 but perhaps we should revert back to installing golang ourselves?

Reply:
fair enough. Let's stick with f33 then.

Reply:
fine with me; in fact I had no issues with f34 (was using that in a first iteration), just wanted to make sure we won't have trouble with a version mismatch down the line